### PR TITLE
Another Vagrant script for Bitbar

### PR DESCRIPTION
**Pull request by axeloz**
Another Vagrant script for Bitbar

### CODE REVIEW

**Comment by manojlds** [comment]
Can you add a screenshot for this?

**Comment by axeloz** [comment]
Here's the screenshot. 

capture d ecran 2016-02-18 a 21 56 32

File: Dev/Vagrant/vagrant.2m.pl [hunk 1]
```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/perl -w
+# <bitbar.title>Vagrant</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author.github>axeloz</bitbar.author.github>
+# <bitbar.author>Axel</bitbar.author>
+# <bitbar.desc>Vagrant status checker.</bitbar.desc>
+#<bitbar.dependencies>perl,vagrant</bitbar.dependencies>
```
Comment (keithamus) [inline Dev/Vagrant/vagrant.2m.pl]: @axeloz do you want to add the screenshot to the metadata here please :smile:

**Comment by axeloz** [comment]
Screenshot added to metadata. 
Screenshot is hosted here: http://hpics.li/095023a

**Pull request merged**